src/carrot-third-party/test.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Should this test data include a tracking pixel?

Reply:
Yes good point, have added one now